pangeo-notebook/packages.txt

```diff
@@ -117,6 +118,7 @@ dask==2024.6.2
 dask-core==2024.6.2
 dask-expr==1.1.6
 dask-gateway==2024.1.0
+dask-geopandas==0.3.1
```

Comment:
Should we try relocking to get `dask-geopandas=0.4.1` (https://github.com/conda-forge/dask-geopandas-feedstock/pull/17)? Or do you want a version tag with 0.3.1 (given all the changes with geopandas 1.0 dropping pygeos and supporting shapely 2.0+ and all that).

Reply:
/condalock